test/test18.py

```diff
@@ -0,0 +1,14 @@
+import sys

```

Comment:
rename to `test/18.py`. since file is already in the test folder, you don't need to put test in the file name